src/logic/__tests__/linkGithubIdentityTest.js

```diff
@@ -0,0 +1,88 @@
+import linkGithubIdentity from '../linkGithubIdentity';
+import {
+  linkGithub,
+  saveCredentialForCurrentUser,
+} from '../../clients/firebase';
+import {getProfileForAuthenticatedUser} from '../../clients/github';
+import {bugsnagClient} from '../../util/bugsnag';
+
+import {
+  credentialFactory,
+  credentialInUseErrorFactory,
+  firebaseErrorFactory,
+  githubProfileFactory,
+  userFactory,
+} from '@factories/clients/firebase';
+
+jest.mock('../../clients/firebase.js');
+jest.mock('../../clients/github.js');
+jest.mock('../../util/bugsnag.js');
+
+describe('linkGithubIdentity', () => {
+  test('success', async() => {
+    const mockCredential = credentialFactory.build();
+    const mockUser = userFactory.build();
+
+    linkGithub.mockResolvedValue({
+      user: mockUser,
+      credential: mockCredential,
+    });
+
+    const {
+      type,
+      payload: {
+        credential: {providerId},
+        user,
+      },
+    } = await linkGithubIdentity.process();
+
+    expect(linkGithub).toHaveBeenCalledWith();
+    expect(saveCredentialForCurrentUser).toHaveBeenCalledWith(mockCredential);
+    expect(type).toBe('IDENTITY_LINKED');
+    expect(providerId).toBe(mockCredential.providerId);
+    expect(user).toEqual(mockUser);
+  });
+
+  test('credential already in use in experimental mode', async() => {
```

Comment:
You can drop the “in experimental mode” from this, that’s an outdated description in the old test.